.github/workflows/cicd.yml

```diff
@@ -0,0 +1,57 @@
+name: CICD
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache pre-commit environment
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/pre-commit
+            ./.pre-commit
+          key: ${{ runner.os }}-precommit-${{ hashFiles('.pre-commit-config.yaml') }}-${{ matrix.python-version }}
+          restore-keys: |
+            ${{ runner.os }}-precommit-${{ matrix.python-version }}-
+            ${{ runner.os }}-precommit-
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: "11" # PySpark works well with Java 8 or 11
```

Comment:
Gebruiken we java/PySpark?

Reply:
verwijderd in https://github.com/woonstadrotterdam/monumenten/pull/1/commits/d8d938f0d87a9eb98935b72dc31a727f67446f58